test/model/src/include/model/kv_transaction.h

```diff
@@ -288,6 +299,7 @@ class kv_transaction {
     /* Transaction information for updates imported from WiredTiger's debug log. */
     txn_id_t _wt_id;
     write_gen_t _wt_base_write_gen;
+    int _wt_ckpt_seq_number;
```

Comment:
```suggestion
    uint64_t _wt_ckpt_seq_number;
```

Reply:
If you decide to proceed with this change, all the `int wt_ckpt_seq_number` should be changed as well.

Reply:
Yes, let's do that.